src/jit/gentree.h

```diff
@@ -4602,14 +4595,11 @@ struct GenTreePutArgStk : public GenTreeUnOp
     // TODO-Throughput: The following information should be obtained from the child
     // block node.
 
-    enum PutArgStkKind : __int8{
-        PutArgStkKindInvalid, PutArgStkKindRepInstr, PutArgStkKindUnroll,
-    };
+    enum PutArgStkKind : __int8{PutArgStkKindInvalid, PutArgStkKindRepInstr, PutArgStkKindUnroll};
 
     PutArgStkKind gtPutArgStkKind;
 
     unsigned gtNumSlots;             // Number of slots for the argument to be passed on stack
-    bool     gtIsStruct;             // This stack arg is a struct.
```

Comment:
(Just a note for future consideration, not really apropos of this change.)

---
In reply to: [82645419](https://github.com/dotnet/coreclr/pull/7537#discussion_r82645419) [](ancestors = 82645419)